recipes/matplotlib/meta.yaml

```diff
@@ -0,0 +1,60 @@
+# Full credit goes to https://github.com/conda/conda-recipes for providing this recipe.
+# It has been subsequently adapted for automated building with conda-forge.
+
+package:
+  name: matplotlib
+  version: 1.5.0rc1
+
+source:
+  fn: matplotlib-1.5.0rc1.tar.gz
+  url: https://github.com/matplotlib/matplotlib/archive/v1.5.0rc1.tar.gz
+
+  patches:
+    # A patch to add Tcl directly to the search path on Windows.
+    - setupext.patch      # [win]
+    # A patch to make Qt4Agg the default backend.
+    - cfg_qt4agg.patch    # [linux]
+    # Patches the matplotlibrc template to default to Qt4.
+    - rctmp_pyside.patch  # [not osx]
+    # Patches setupext to look in PREFIX for the tcl/tk headers.
+    - osx-tk.patch        # [osx]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy *
+    - python-dateutil
+    - freetype
+    - msinttypes          # [win]
+    - cycler
+    - nose
+    - pyparsing
+    - pytz
+#    - py2cairo            # [linux and py2k]
```

Comment:
I'm currently running into a similar problem: could it be that obvious-ci does not set CONDA_PY when calling the conda build procedure? My observation was that the wrong py versions are set so this selector is only right when the "outer" conda version is the same as the one which should build the thing...

It works on windows, as there `CONDA_PY` is set via the build matrix.